CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ if(NOT Poco_FOUND)
   if(WIN32)
     list(APPEND POCO_CXX_FLAGS "/wd4244 /wd4530 /wd4577")
   else()
-    list(APPEND extra_cmake_args "-DCMAKE_C_FLAGS=-Wno-shift-negative-value")
+    list(APPEND extra_cmake_args "-DCMAKE_C_FLAGS=${CMAKE_C_FLAGS} -Wno-shift-negative-value")
```

Comment:
Is this meant to be `CMAKE_CXX_FLAGS`?

Reply:
Possibly but it was reviewed and merged verbatim in #28

Reply:
OK, just checking because the PR title says CXX.